desktop-exporter/app/utils/array-to-tree.ts

```diff
@@ -99,14 +106,6 @@ function getEarliestStartTime(children: TreeItem[]): number {
   }
 
   let startTimes = children.map((treeItem) => {
-    if (treeItem.status === SpanDataStatus.missing) {
-      throw new Error(
-        // This should also happen in this implementation, since that the child span
-        // must have SpanData (minimally a parentSpanID) in order for the parent span to be created.
-        "Unexpected type: A child of a 'missing' parent span appears to have no SpanData.",
-      );
-    }
-
```

Comment:
Did I make the types more complicated just to get rid of this assertion? Yes... yes I did